bot/src/main/java/com/almightyalpaca/discord/jdabutler/commands/commands/ChangelogCommand.java

```diff
@@ -67,11 +77,33 @@ public void dispatch(final User sender, final TextChannel channel, final Message
                 reply(event, "The specified version does not exist");
                 return;
             }
+
+            // Get time of build
+            String publishedTime;
+            try
+            {
+                final JenkinsBuild build = args == null
+                        ? JENKINS.getLastSuccessfulBuild()
+                        : JENKINS.getBuild(Integer.parseInt(args[0]));
```

Comment:
I believe I made a tiny mistake here. Shouldn't `args[0]` not be the item name according to line 45?

I mean `VersionedItem#getChangelogProvider` always returns `null` at the moment so it will always stop executing with `"No Changelogs set up for " + item.getName()` (line 57) but if this is gonna be changed, this will crash when providing another Item since `args[0]` isn't a build number / an integer.

I would fix that by replacing `args[0]` with `args[args.length - 1]`.

Reply:
@kantenkugel I talked already with Alpaca in the [JDA Server](https://discordapp.com/channels/125227483518861312/125227483518861312/609133986073280515) about this today, he has problems with the general code base right now so it would be great if you can also look over it (and maybe you can give me some response on [that one](#pullrequestreview-260090308) too). However, it "sounds plausible" for Alpaca and it's ready to be merged (after I updated the code based on my last review).